plugins/org.eclipse.n4js.ui/src/org/eclipse/n4js/ui/wizard/dependencies/ExternalLibrariesInstallHelper.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2018 NumberFour AG.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   NumberFour AG - Initial API and implementation
+ */
+package org.eclipse.n4js.ui.wizard.dependencies;
+
+import java.util.Map;
+
+import org.eclipse.core.runtime.MultiStatus;
+import org.eclipse.core.runtime.SubMonitor;
+import org.eclipse.n4js.ui.external.ExternalLibrariesActionsHelper;
+
+import com.google.inject.Inject;
+
+/**
+ * Helper for installing npm dependencies.
+ */
+public class ExternalLibrariesInstallHelper {
+
+	@Inject
+	private ProjectDependenciesHelper dependneciesHelper;
```

Comment:
can you also adjust the dictionary regarding `dependNecies`?

Reply:
good idea